### PR TITLE
add confirm contract modal && add horizontal-dl component && move wri…

### LINKED ISSUE #232: [wallet] contract: P2 2.9999 Modal - Confirm Contract Transaction

### PR DESCRIPTION
**Pull request by anshuomomo**
…te contract to storybook

**What type of PR is this?**
> /kind ui-change

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #232 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by anshuomomo** [comment]
PC && mobile screenshot

**Comment by anshuomomo** [comment]
`For the component horizontal-table.  test by using query={GET_ACTIONS_BY_HASH}.`

File: .storybook/confirm-contract-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      {
+        title: "Data",
+        dataIndex: "data"
+      }
+    ];
+  }
+
+  public render(): JSX.Element {
+    const { showModal } = this.props;
+    const dataSource = {
```
Comment (puncsky) [inline .storybook/confirm-contract-modal.tsx]: take them from input props?

File: .storybook/confirm-contract-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+// @ts-ignore
+import { t } from "onefx/lib/iso-i18n";
+import React from "react";
+import { Board } from "../src/shared/common/board";
+import { Column, HorizontalTable } from "../src/shared/common/horizontal-table";
+import { ModalBody } from "../src/shared/common/modal-body";
+
+export interface Props {
+  showModal: boolean;
+  generateTransaction: Function;
```
Comment (puncsky) [inline .storybook/confirm-contract-modal.tsx]: add some comments how to use it?

**Pull request merged**